dali/python/nvidia/dali/pipeline.py

```diff
@@ -135,13 +136,34 @@ class Pipeline(object):
     you will need to call :meth:`start_py_workers` before calling :meth:`build` of any
     of the pipelines. You can find more details and caveats of both methods in Python's
     ``multiprocessing`` module documentation.
+`py_callback_pickler` : module or tuple, default = nvidia.dali.pickling
+    If `py_start_method` is set to *spawn* callback passed to parallel ExternalSource must be picklable.
+    If run in Python3.8 or newer, DALI uses customized pickle (`nvidia.dali.pickling`) when
+    serializing callbacks to support serialization of local functions and lambdas.
+
+    However, if you need to serialize more complex objects like local classes or you are running
+    older version of Python you can provide external serialization package such as dill or cloudpickle
```

Comment:
For me it wasn't immediately clear that I can pass `dill` _module_ as an argument. How about adding an example?

Reply:
I've added the example in the notebook with parallel External Source spawn use cases. Do you think abut explicitly putting some code into the docstring too?

Reply:
done

Reply:
done